push_notifications/gcm.py

```diff
@@ -119,6 +120,8 @@ def _cm_send_request(
 
 	payload = {"registration_ids": registration_ids} if registration_ids else {}
 
+	data = copy.copy(data)
```

Comment:
There is a `dict.copy` method specifically for this. You don't need to import copy.